### DISCUSSION TITLE
Split by language should offer options for org languages?

### DISCUSSION THREAD

**Discussion by nicpottier** [nyaruka/floweditor#1115] [Ideas]
See: https://app.intercom.io/a/apps/l6e7tr01/conversations/25036819776

Seems that we could maybe show the org languages in a drop down here to make this a bit easier since expecting users to understand iso codes is a big ask.

**Comment by ericnewcomer** [comment]
Do we want a distinct split by language modeled after split by group membership?

**Comment by nicpottier** [comment]
As in they specify the options they care about?

Maybe we should super dumb it down and there are no options at all, it just creates an exit for every language configured on their org. I can only think of one org that has more than 4 languages (U-Report Global). Suppose that one could get out of control.

But ya, totally fine doing it where they use a select2 like thing to add all the languages they care to split on and we create exits for each of those.

**Comment by ericnewcomer** [comment]
Ya primary question there is whether it should have a bespoke split which I believe the answer there is yes. Adding languages to split on feels the safest, I think it's quite common for them to have excess languages on their account.
